cli/ts/index.ts

```diff
@@ -1,3 +1,4 @@
+#!/usr/bin/env node
```

Comment:
Because when you distribute it as a npm package, it will try to run as a bash script if this is missing. you can try by running `npm link` inside the cli folder

Reply:
Ok, good!